wtxmgr/tx.go

```diff
@@ -670,11 +670,20 @@ func (s *Store) rollback(ns walletdb.ReadWriteBucket, height int32) error {
 
 	for _, op := range coinBaseCredits {
 		opKey := canonicalOutPoint(&op.Hash, op.Index)
-		unminedKey := existsRawUnminedInput(ns, opKey)
-		if unminedKey != nil {
-			unminedVal := existsRawUnmined(ns, unminedKey)
+		unminedSpendTxHashKeys := fetchUnminedInputSpendTxHashes(ns, opKey)
+		for _, unminedSpendTxHashKey := range unminedSpendTxHashKeys {
+			unminedVal := existsRawUnmined(ns, unminedSpendTxHashKey[:])
+			// It might be the case that this spending transaction
+			// spent multiple outpoints from the previous
+			// transaction output. If so, then we won't find this
```

Comment:
Multiple points from the previous transaction output? That's not possible in this context afaict. Guessing you mean spends multiple outputs from the previous transaction, so this transaction will have already been removed after the first spent output.

Reply:
Fixed, not sure where I got this wording from lol...